lib/hammer_cli_csv/subscriptions.rb

```diff
@@ -46,7 +46,9 @@ def export_subscriptions(csv, organization)
           quantity = subscription['quantity'] < 0 ? "Unlimited" : subscription['quantity']
           csv << [name, organization['name'], nil, subscription['name'],
                   quantity, subscription['product_id'], subscription['contract_number'],
-                  subscription['account_number']]
+                  subscription['account_number'],
+                  DateTime.parse(subscription['start_date']).strftime('%m/%d/%Y'),
+                  DateTime.parse(subscription['end_date']).strftime('%m/%d/%Y')]
```

Comment:
Is m/d/y enough information? Doesn't this parse and format remove information from this field (i.e. timezone, time stamp, etc.)? Here's a start_date from one of my Katello::Pool objects for reference: 2016-07-27T18:13:03.540+0000